pkg/cluster/context.go

```diff
@@ -178,6 +178,15 @@ func (c *Context) Delete() error {
 	if err != nil {
 		return fmt.Errorf("error listing nodes: %v", err)
 	}
+
+	// clean up the kind kube config file that was created earlier
+	// also let the user know that they need to unset $KUBECONFIG to
+	// go back to their normal contexts
+	os.Remove(c.KubeConfigPath())
```

Comment:
i will leave it to @BenTheElder for the decision on cleanup here.
my vote leans towards not doing it by default, but having an explicit CLI flag for it.

Reply:
sounds interesting, why do you think it shouldn't be done by default?